Kudu.Services/SiteExtensions/SiteExtensionController.cs

```diff
@@ -55,18 +57,18 @@ public SiteExtensionController(ISiteExtensionManager manager)
         }
 
         [HttpPut]
-        public async Task<SiteExtensionInfo> InstallExtension(string id, SiteExtensionInfo requestInfo)
+        public async Task<HttpResponseMessage> InstallExtension(string id, SiteExtensionInfo requestInfo)
         {
             if (requestInfo == null)
             {
                 requestInfo = new SiteExtensionInfo();
             }
 
-            SiteExtensionInfo extension;
+            Tuple<bool, SiteExtensionInfo> result;
```

Comment:
Is it possible to get away from tuple (hard to read)?   for instance, can we deduct this from SiteExtensionInfo.
  - call from ARM
  - info indicates success
  - info indicate installed time > request start time